<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="qLabelSourceText">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Исходный текст</string>
   </property>
  </widget>
  <widget class="QLabel" name="qLabelOutputText">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Результат</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="qTextEditSourceText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>280</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="qTextEditOutputText">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>280</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="qPushButtonSolver">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>75</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>ВЫПОЛНИТЬ
ЗАДАНИЕ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qPushButtonClear">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>75</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>ОЧИСТИТЬ
ФОРМЫ</string>
   </property>
  </widget>
  <widget class="QLabel" name="qLabelRequirement">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>531</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Дан тест. Найти в нем те слова, которые начинаются и заканчиваются одной буквой</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="Line" name="horizontalLine">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>581</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
